topbeat/tests/system/test_filesystem.py

```diff
@@ -17,8 +17,8 @@ def test_filesystems(self):
             process_stats=False,
             filesystem_stats=True
         )
-        topbeat = self.start_topbeat()
-        self.wait_until(lambda: self.output_has(lines=1))
+        topbeat = self.start_beat()
+        self.wait_until(lambda: self.log_contains(msg="output worker: publish"))
```

Comment:
The reason I switched was that it seems sometime my local machine was too fast, and the check for lines=1 never hit, as there were already more lines then one. Best would be a check lines >= 1.